heat/core/logical.py

```diff
@@ -106,7 +106,11 @@ def local_all(t, *args, **kwargs):
 
 
 def allclose(
-    x: DNDarray, y: DNDarray, rtol: float = 1e-05, atol: float = 1e-08, equal_nan: bool = False
+    x: DNDarray,
+    y: DNDarray,
+    rtol: Optional[float] = None,
```

Comment:
I think this needs to be discussed (e.g. in the PR talk): should we keep the NumPy/PyTorch-standard and choose some float limits as default (but adapted to Heats float32-standard datatype) or do we want to proceed differently and let tolerances by default be chosen automatically depending on the input datatypes?